src/redisraft.h

```diff
@@ -625,6 +631,9 @@ typedef struct RaftReq {
     int type;
     RedisModuleBlockedClient *client;
     RedisModuleCtx *ctx;
+    RedisModuleTimerID timeout_timer;
+    raft_index_t raft_idx;
+    unsigned long long client_id;
```

Comment:
```suggestion
    raft_session_t client_id;
```

Reply:
not needed yet, so just removing for now